src/components/Dashboard/EventCard.vue

```diff
@@ -152,12 +169,11 @@ function handleJoin({ call = false } = {}) {
 			</template>
 		</p>
 		<span class="event-card__room secondary_text">
-			<span class="event-card__room-prefix">
-				{{ props.eventRoom.roomType === CONVERSATION.TYPE.ONE_TO_ONE ? t('spreed', 'With') : t('spreed', 'In') }}
-			</span>
 			<NcChip type="tertiary"
-				no-close
-				:text="props.eventRoom.roomDisplayName">
+				:text="props.eventRoom.roomType === CONVERSATION.TYPE.ONE_TO_ONE
```

Comment:
It's  a bit too much for the template. Let's move to computed and add TRANSLATORS comments for this